Makefile

```diff
@@ -21,9 +21,11 @@ install-share: transpile
 	mkdir -p $(PREFIX)/share/splatsh/node_modules
 	node -e 'console.log(JSON.stringify({ dependencies: require("./package.json").dependencies }))' > $(PREFIX)/share/splatsh/package.json
 	npm i --only=production --prefix $(PREFIX)/share/splatsh
-	# this should be install but idk how to do dirs lol
-	cp -r build/* $(PREFIX)/share/splatsh/
+	ln -s build $(PREFIX)/share/splatsh
```

Comment:
why the symlink? this should be a copy so you can safely delete the git clone

Reply:
Good point you have there.